docker/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:14.04
+
+MAINTAINER dongjie.shi dongjie.shi@intel.com
```

Comment:
MAINTAINER The BigDL Authors https://github.com/intel-analytics/BigDL

Reply:
OK.